CRM/PCP/Form/PCPAccount.php

```diff
@@ -115,6 +115,33 @@ public function setDefaultValues() {
       }
 
       CRM_Core_BAO_UFGroup::setProfileDefaults($this->_contactID, $fields, $this->_defaults);
+
+      if (!empty($this->_defaults['image_URL'])) {
+        $this->assign("imageURL", CRM_Utils_File::getImageURL($this->_defaults['image_URL']));
+        $this->removeFileRequiredRules('image_URL');
+
+        $deleteExtra = json_encode(ts('Are you sure you want to delete contact image.'));
```

Comment:
you might want to change . to ?

Reply:
```
$ grep -r "Are you sure you want to delete" | grep image
CRM/Core/BAO/UFGroup.php:      $deleteExtra = json_encode(ts('Are you sure you want to delete contact image.'));
CRM/Contact/Form/Contact.php:      $deleteExtra = json_encode(ts('Are you sure you want to delete contact image.'));
```

This is a dozen lines that are present in 2 other places. Could you refactor?

Similar to what Yashodha recommends, and since it will mean updating translations, I recommend adding "the" image ("Are you sure you want to delete the contact image?")

Reply:
Thank you both, will refactor and update the PR soon.